docs/content/en/docs/reference/artifacts.md

```diff
@@ -157,7 +165,7 @@ govc library.create "CodeBuild"
 `image-builder/images/capi/packer/ova/ubuntu-2004.json`
 9. Setup image-builder and run the OVA build for the Kubernetes version.
 ```
-make release-ova-ubuntu-2004-1-21
+RELEASE_BRANCH=1-22 make release-ova-ubuntu-2004
```

Comment:
Did you meant to put RELEASE_BRANCH and make on the same line?

Reply:
That sets that env var for the context of that one command. Otherwise, you need to export it as well

Reply:
Yup this was done on purpose

Reply:
Thanks! That was my only question.